runtime/queries/nix/highlights.scm

```diff
@@ -17,6 +17,8 @@
   "with" 
 ] @keyword
 
+(variable_expression name: (identifier) @variable)
```

Comment:
We also need to move these two patterns up in the file, right after this pattern :point_up::

```scm
(select_expression
  attrpath: (attrpath attr: (identifier)) @variable.other.member)

(apply_expression
  function: [
    (variable_expression name: (identifier) @function)
    (select_expression
      attrpath: (attrpath
        attr: (identifier) @function .))])
```

For example in the `flake.nix` in this repo it fixes the highlighting for `import nixpkgs` (the `apply_expression` pattern) and `builtin.path` (the `select_expression` papttern)

Reply:
Fixed, thanks!